.pullapprove.yml

```diff
@@ -0,0 +1,9 @@
+approve_by_comment: true
+approve_regex: '^(Approved|lgtm|LGTM|:shipit:|:star:|:\+1:|:ship:)'
+reject_regex: ^Rejected
+reset_on_push: false
+reviewers:
+  teams:
+  - image-maintainers
+  name: default
+  required: 2
```

Comment:
we are 3 maintainers right now, me you and brandon, 2 is the minimum consensus to make a PR ready to merge